app/assets/javascripts/discourse/app/instance-initializers/markdown-tabs.js

```diff
@@ -0,0 +1,50 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+
+function initializeMarkdownTabs(api) {
+  api.decorateCooked(
+    ($elem) => {
```

Comment:
We should use the non-jquery version of the API

```
api.decorateCookedElement((element) => {
```
(which reminds me... we should probably think about formally deprecating the jquery-based one...)